modules/electrophysiology_browser/php/eeg_visualization.class.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
```

Comment:
All this should be altered to match the normal way we declare PHP class and file docs. it won't pass Travis.